plugins/inputs/http_response/README.md

```diff
@@ -45,6 +45,9 @@ This input plugin checks HTTP/HTTPS connections.
   # response_string_match = "ok"
   # response_string_match = "\".*_status\".?:.?\"up\""
 
+  ## Optional status code match of the response
+  # response_status_code_match = 204
```

Comment:
The name of the setting is a little confusing.  It is similar to the previous setting "response_string_match" but in that case you're doing a regex match.  I think calling this new setting response_status_code or response_expected_status_code would be clearer.

The convention with config examples is to show the default value in the line that's commented out.  It looks like the new setting defaults to 0.


Could you also add in the config example some text describing how the setting works?  Something like this:
```suggestion
  ## Expected response status code.
  ## The status code of the response is compared to this value.  If they match, the field
  ## "response_status_code_match" will be 1, otherwise it will be 0.  If the
  ## expected status code is 0, the check is disabled and the field won't be added.
  # response_status_code = 0
```